src/Chapter3.hs

```diff
@@ -971,6 +1129,30 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data Weekday = Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday deriving(Show, Enum, Eq)
+
+class Weekdays a where
+  isWeekend :: a -> Bool
+  nextDay :: a -> a
+  daysToParty :: a -> Int
```

Comment:
Introducing a separate typeclass wasn't required in this task, but it's actually really great that you did it! I can see that you understand typeclasses perfectly :100: Also, this typeclass will become handy, if you decide to implement weekdays in the American system, so you will have two instances for two different data types, but with they both will have the same interface.